### DISCUSSION TITLE
Tooltip doesn't show on hover?

### DISCUSSION THREAD

**Discussion by geoidesic** [carbon-design-system/carbon-components-svelte#1802] [Q&A] [answered]
https://carbon-components-svelte.onrender.com/components/Tooltip

It seems ironic that the tooltips on the docs page (like for the Copy Code icon) work the way tooltips are expected to work on a desktop browser (i.e. display on hover) but the Tooltip component that the docs describe doesn't seem to offer that functionality :)

Currently I'm not sure how this component is different from a button?

I can appreciate that on click is useful and necessary for mobile devices but it's not useful for desktop, where the idea is that a tooltip provides information about the element you are hovering over.

Am I doing it wrong? Is there a way to get it to show the text on hover? 

The one on the IBM live demo works as expected: https://carbondesignsystem.com/components/tooltip/usage/#live-demo

**Comment by metonym** [comment, marked as answer]
Fixed in [v0.101.1](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.101.1)
